pages/1.12/storage/partner integration/index.md

```diff
@@ -0,0 +1,105 @@
+# Partner Integrations
+
+## Portworx
+
+Portworx solves the operational and data management problems enterprises encounter when running stateful applications on DC/OS. 
+Using Portworx volumes, you can:
+
+* Dynamically create volumes for tasks at run time, no more submitting tickets for storage provisioning
+* Dynamically and automatically resize volumes based on demand while task is running
+* Run tasks on the same hosts that your data is located on for optimum performance
+* Avoid pinning services to particular hosts, reducing the value of automated scheduling
+* Avoid fragile block device mount/unmount operations that block or delay failover operations
+* Encrypt data at rest and in flight at the container level
+
+## Installing Portworx with DC/OS
+
+It is fairly straightforward to install Portworx with DC/OS. 
```

Comment:
No. Do not send our readers away from the DC/OS website unless absolutely necessary. Please rewrite this section to include comprehensive installation instructions, with illustrations such as the ones from the Portworx DCOS documents.

Reply:
It is duplication of all the information and the format won't be easy to merge. Also, it is not just one page but many such pages for Portworx and DCOS integration.